src/index.html

```diff
@@ -7,11 +7,150 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
+    <title>HTML Form</title>
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-fieldset">
+        <legend>Personal information</legend>
+        <label class="form-input">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-input">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-input">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="form-input">
+          Full date of birth:
+          <input
+            type="date"
+            name="birth"
+          >
+        </label>
+        <label class="form-input">I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>Registration</legend>
+        <label class="form-input">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form-input">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="30"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-input">
+          Do you love cats?
+          <div>
+            <input
+            type="radio"
+            name="cats"
+            > Yes
+          </div>
+          <div>
+            <input
+            type="radio"
+            name="cats"
+            > No
+          </div>
+        </label>
```

Comment:
Each radion-input should have its own label. Now if I click on `no`, `yes` becomes selected instead of `no`.